Makefile

```diff
@@ -61,6 +61,7 @@ tscheck:
 	$(MAKEJS) tscheck
 
 lint-ci: lint check-compat-data-ci
+	$(NODE) scripts/generators/readmes.js
```

Comment:
Can we add a new makefile task? e.g. `make generate-readme`. As it does not seem that it should be a lint task.